osprey_flask_app/utils.py

```diff
@@ -59,3 +63,48 @@ def create_full_arg_dict(args):
 
             arg_dict[arg] = args.get(arg, default=default)
     return arg_dict
+
+
+def validate_inputs(arg_dict):
+    """Check that inputs have valid values and that filepaths exist.
+    Parameters
+        1. arg_dict (dict): arguments supplied to osprey with corresponding values
```

Comment:
Can you be more specific about what the `arg_dict` should contain and what constitutes "validity"?